### PR TITLE
test: add cypress

### PR DESCRIPTION
**Pull request by pyphilia**
ref #7

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        shell: bash
+        env:
+          VITE_PORT: ${{ vars.VITE_PORT }}
+          VITE_VERSION: ${{ vars.VITE_VERSION }}
+          VITE_GRAASP_DOMAIN: ${{ vars.VITE_GRAASP_DOMAIN }}
+          VITE_GRAASP_API_HOST: ${{ vars.VITE_GRAASP_API_HOST }}
+          VITE_GRAASP_AUTH_HOST: ${{ vars.VITE_GRAASP_AUTH_HOST }}
+          VITE_GRAASP_PLAYER_HOST: ${{ vars.VITE_GRAASP_PLAYER_HOST }}
+          VITE_GRAASP_LIBRARY_HOST: ${{ vars.VITE_GRAASP_LIBRARY_HOST }}
+          VITE_GRAASP_ANALYZER_HOST: ${{ vars.VITE_GRAASP_ANALYZER_HOST }}
```
Comment (spaenleh) [inline .github/workflows/cypress.yml:46]: Are all of these necessary ?

File: cypress/fixtures/members.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { MemberFactory } from '@graasp/sdk';
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (spaenleh) [inline cypress/fixtures/members.ts:3]: Remove ?

**Review by spaenleh — APPROVED** [review]
Looks good ! Thank you !

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (pyphilia) [reply .github/workflows/cypress.yml:46]: For platform switch I guess, but yeah not everything is necessary but they will at some point I think.

Comment (spaenleh) [reply .github/workflows/cypress.yml:46]: @pyphilia Ah right !

**Pull request merged**